### DISCUSSION TITLE
Switcher change on list table

### DISCUSSION THREAD

**Discussion by hoangnamitc** [orchidsoftware/platform#2341] [Q&A] [answered]
Hello,

I have table:
<img width="534" alt="image" src="https://user-images.githubusercontent.com/7042462/185757186-0721e0b1-3861-4d70-b346-c260b11101bf.png">

My Code:
![image](https://user-images.githubusercontent.com/7042462/185757233-bb99182f-59db-4db9-b7db-70adce6cfb12.png)

How to change Switcher with change on database?

Thanks

**Comment by hexbyteinc** [comment]
thanks

**Comment by tabuna** [comment, marked as answer]
Hi @hoangnamitc You can do it like this:

```php
  ->render(function (Client $client) {
      return CheckBox::make("users[$client->id]")
          ->sendTrueOrFalse()
          ->value($client->active);
  }),
```

Then by clicking on the button for example save in the `commandBar`, you will be able to get a list of clients and their status for mass updating. If you want to make it a js ajax request, then you can do it yourself by adding your own [stumulus controller for example](https://orchid.software/en/docs/javascript/).

**Reply by hoangnamitc** [reply]
Thanks for answer !
